### PR TITLE
Detect new brands: Diva, Oale, Zentality, Black Bear, OpelMobile Fortis and Detect devices for existing brands: BQ, Sky, Blu, Alfawise, TWM, Mobo, Jinga, iHunt, Alcatel, Teclast, Lenovo, AllDocube, Wiko, iSWAG, Zuum, LG, Mediacom, Samsung, Huawei, Maxwest, Elephone, Vorke

### PR DESCRIPTION
**Pull request by sanchezzzhak**
this PR merge after #6554

### CODE REVIEW

File: regexes/device/mobiles.yml [hunk 1]
```diff
@@ -7820,7 +7847,7 @@ K-Touch:
 
 # kyocera (kyocera.co.jp, kyocera.com)
 Kyocera:
-  regex: 'Kyocera|KWC-|QC-|(?:NP501KC|302KC|503KC|602KC|705KC|901KC|C51[27]0|C5155|C5215|C67[35]0|C6522N?|C6530N|C6740N?|C6743|E6[89]10|KYL2[13]|KYV3[13-9]|KYT3[123]|KYF3[123579]|KYX31|KYY2[1-4]|S2151|KC-S701|S4-KC|KYV40U?|KYV4[1-8]|KYV4[47]|KC-S301AE|X3-KC|S6-KC|E6560[CT]|E6782[L]?|E6790(?:TM)?|[47]04KC|A001KC)(?:[-_]u2?)?(?:[);/ ]|$)'
```
Comment (sgiehl) [inline regexes/device/mobiles.yml:7823]: Wouldn't that remove detections for useragents like
`QC-2235/1.0.52 UP.Browser/4.1.22b1` (event though the detection might currently not be correct, as it removes the `QC`)

Comment (sgiehl) [reply regexes/device/mobiles.yml:7823]: Just looked up a useragent containing QC 🙈 Might be good if we would still detect them if possible

Comment (sanchezzzhak) [reply regexes/device/mobiles.yml:7823]: @sgiehl 
added tests for devices `QC-[0-9]{4}`, which I managed to find from the list
https://deviceatlas.com/device-data/devices/kyocera

File: regexes/device/mobiles.yml [hunk 2]
```diff
@@ -7957,7 +7990,7 @@ Kyocera:
       model: 'KZ $1'
     - regex: 'Kyocera(?:[\-/])?([a-z0-9]+)'
       model: '$1'
-    - regex: '(?:KWC|QC)-([a-z0-9]+)'
+    - regex: '(?:KWC|QC)-?([a-z0-9]+)'
       model: '$1'
```
Comment (sgiehl) [inline regexes/device/mobiles.yml:7994]: Shouldn't the model names be prefixed with `QC ` or `KWC ` or are the model names numeric only?

Comment (sanchezzzhak) [reply regexes/device/mobiles.yml:7994]: Yes, they should, I will fix it

**Review by sgiehl — APPROVED** [review]

**Pull request merged**